### DISCUSSION TITLE
Security Warning! The gradle wrapper ~ could be malicious.

### DISCUSSION THREAD

**Discussion by tgkei** [mfussenegger/nvim-jdtls#534] [Q&A]
Hello.
There is a warning when trying to open java file with specific gradle wrapper. It says add following command
`{"sha256": "{some string}",
"allowed": true}`

I added option as https://github.com/mfussenegger/nvim-jdtls/discussions/377 this issue, but still have same warning and lsp does not work properly. My option is following

``` lua
local config = {
  init_options = {
    -- Here you can configure eclipse.jdt.ls specific settings
    -- See https://github.com/eclipse/eclipse.jdt.ls/wiki/Running-the-JAVA-LS-server-from-the-command-line#initialize-request
    -- for a list of options
    settings = {
      java = {
          import = { -- imports doesn't work either
            gradle = {
                enabled = true,
                wrapper = {
                    enabled = true,
                    checksums = {
                        {
                            sha256  = '{sha from warning code}',
                            allowed = true
                        }
                    },
                }
            }
        },
      },
    },
    bundles = {}
  },
}
```

can please anyone help?

**Comment by tgkei** [comment]
still nothing?

**Comment by mgallegoa** [comment]
Hi @tgkei ,

Same issue here, how do you solved it?

**Reply by mgallegoa** [reply]
Hi @tgkei ,

Solved with the next config in the NVIM_PATH/ftplugin/java.lua file:
https://github.com/exosyphon/nvim/blob/0aa48126c7f35f2009c5a695860a53c8a450485f/ftplugin/java.lua#L1

**Reply by sxhmilyoyo** [reply]
Hi, @mgallegoa , do you know which part of the provided file resolved this issue? Thanks.
